docs/en/guides/Plugin-test.md

```diff
@@ -294,6 +294,48 @@ The verify description for SegmentRef
 | networkAddress | The peer value of parent exit span.
 | refType | Ref type, options, CrossProcess or CrossThread.
 
+**Meter verify description format**
+```yml
+meterItems:
+-
+  serviceName: SERVICE_NAME(string)
+  meterSize: SEGMENT_SIZE(int)
+  meters:
+  - ...
+```
```

Comment:
I think you missed this part of the document for this, https://github.com/apache/skywalking/blob/dba60957d9cf55a5bdf8c50eaeb9be76eeb401b9/docs/en/guides/Plugin-test.md#the-example-process-of-writing-expected-data